auth-tokens/src/main/java/com/palantir/tokens/auth/UnverifiedJsonWebToken.java

```diff
@@ -145,34 +152,21 @@ private static JwtPayload extractPayload(String payload) {
         }
     }
 
-    /**
-     * Returns an encoded UUID from a length 16 byte array.
-     * <p>
-     * Palantir stores UUIDs in this format to optimize on shorter JWTs.
-     */
-    private static String decodeUuidBytes(byte[] bytes) {
-        Preconditions.checkArgument(
-                bytes.length == 16,
-                "Invalid JWT: cannot decode UUID, require 16 bytes",
-                SafeArg.of("bytesLength", bytes.length));
-        ByteBuffer byteBuffer = ByteBuffer.wrap(bytes);
-        long high = byteBuffer.getLong();
-        long low = byteBuffer.getLong();
-        return UuidStringConverter.toString(new UUID(high, low));
-    }
```

Comment:
Jackson's UUID deserializer support deserializing Base64-encoded UUIDs, so we don't need this. This avoids some additional allocations during deserailizing.